oaknational/Oak-Web-Application
# feat: adds legacy button to school picker CUR-248

## Description

- adds link to teacher hub page to school picker

## Issue(s)

Fixes #CUR-248

## How to test

1. Go to https://deploy-preview-1888--oak-web-application.netlify.thenational.academy/beta/teachers/curriculum
2. Click on school picker
3. You should see link to legacy downloads page

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:

<img width="798" alt="Screenshot 2023-09-08 at 12 03 35" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/aed0afcf-89dc-4e2c-8469-08cbe01a3916">

## Checklist
